CRM/Dashlet/Page/GettingStarted.php

```diff
@@ -87,46 +87,37 @@ public function run() {
    * @return array
    */
   private function _gettingStarted() {
-    // Fetch data from cache
-    $cache = CRM_Core_DAO::executeQuery("SELECT data, created_date FROM civicrm_cache
-      WHERE group_name = 'dashboard' AND path = 'gettingStarted'");
-    if ($cache->fetch()) {
-      $expire = time() - (60 * 60 * 24 * self::CACHE_DAYS);
-      // Refresh data after CACHE_DAYS
-      if (strtotime($cache->created_date) < $expire) {
-        $new_data = $this->_getHtml($this->gettingStartedUrl());
-        // If fetching the new html was successful, return it
-        // Otherwise use the old cached data - it's better than nothing
-        if ($new_data) {
-          return $new_data;
-        }
+    $value = Civi::cache('community_messages')->get('dashboard_gettingStarted');
```

Comment:
@totten is us of the 'community_messages' here correct? I'm confused about whether you can just add extra caches this way

Reply:
@mlutfy ah it was already defined - that's the missing piece for me